Makefile

```diff
@@ -184,6 +184,9 @@ clean:
 	@rm -rf test/e2e/log
 	@rm -rf e2e.namespace
 
+lint:
+	find . -name '*.go' -not -path "./vendor/*" | xargs gofmt -w
+	find . -name '*.go' -not -path "./vendor/*" | xargs goimports -w
```

Comment:
This likely isn't the greatest structure but I didn't have an immediate answer on how to clean it up.

It looks like we do some combination of the following:
- Not run both of those checks
- Make the list of files spit out by that `find` command a prerequisite and update the xargs command to run both gofmt/goimports references that same set of files.